README.md

```diff
@@ -177,20 +177,22 @@ OnePlus | 9 | lemonade | | under development
 
 </details>
 
-<details><summary><b>Xiaomi</b></summary>
+<details><summary><b>Xiaomi & Poco</b></summary>
```

Comment:
Shouldn't we just keep the name Xiaomi?

Or write it as `Xiaomi / Poco / Redmi` or something?

Reply:
Usually, the brand "Redmi" is preceded by "Xiaomi", which is not the case of Poco as far I know, but as you want !

Reply:
Then lets take `Xiaomi / Poco` :slightly_smiling_face: